net.sourceforge.liferea.json

```diff
@@ -0,0 +1,63 @@
+{
+    "app-id":"net.sourceforge.liferea",
```

Comment:
```suggestion
    "app-id":"de.lzone.Liferea",
```

Reply:
app-id is net.sourceforge.liferea: https://github.com/lwindolf/liferea/blob/master/src/liferea_application.c#L345 or https://github.com/lwindolf/liferea/blob/master/liferea.appdata.xml.in#L3